www/src/pages/en/usage/env-variables.md

```diff
@@ -85,7 +85,7 @@ const dbUrl = env.DATABASE_URL;
 
 Since the default `.env` file is not committed to version control, we have also included a `.env.example` file, in which you can optionally keep a copy of your `.env` file with any secrets removed. This is not required, but we recommend keeping the example up to date to make it as easy as possible for contributors to get started with their environment.
 
-Some frameworks and build tools, like Next.js, suggest that you store secrets an `.env.local` file and commit `.env` files to your project. This is not recommended, as it could make it easy to accidentally commit secrets to your project. Instead, we recommend that you store secrets in `.env`, keep your `.env` file in your `.gitignore` and only commit `.env.example` files to your project.
+Some frameworks and build tools, like Next.js, suggest that you store secrets in `.env.local` file and commit `.env` files to your project. This is not recommended, as it could make it easy to accidentally commit secrets to your project. Instead, we recommend that you store secrets in `.env`, keep your `.env` file in your `.gitignore` and only commit `.env.example` files to your project.
```

Comment:
still wrong?

```suggestion
Some frameworks and build tools, like Next.js, suggest that you store secrets in `.env.local` file and commit `.env` files to your project. This is not recommended, as it could make it easy to accidentally commit secrets to your project. Instead, we recommend that you store secrets in an `.env`, keep your `.env` file in your `.gitignore` and only commit `.env.example` files to your project.
```

Reply:
oops, mb. Using `a` instead of `an` since we call it "dotenv". For reference, [docker docs](https://docs.docker.com/compose/environment-variables/) uses `a`.

![image](https://user-images.githubusercontent.com/89210438/204134030-0a6fa649-42de-44cf-8af6-2b79e8a743d8.png)